src/components/SparklineChart/index.tsx

```diff
@@ -58,10 +59,15 @@ export const SparklineChart: React.FC<ChartProps> = ({
   >("rgba(0,245,228,1)");
   const [borderColor, setBorderColor] = useState<string>("rgba(0,245,228,1)");
 
-  const [border, setBorder] = useState<BorderObject>({
-    color: "transparent",
-    width: 0,
-  });
+  const { theme, setTheme } = useTheme();
+
+  const getChartBackground = () => {
+    if (theme === "dark") {
+      return "rgba(24,24,38,1)";
+    } else {
+      return "rgba(255, 255, 255, 1.0)";
+    }
```

Comment:
I think I saw this in another PR

Reply:
Very close to it in the ModularChart PR, I'm trying to re-use as much code as possible but with dynamic-rendering limits on tw-style-components I had to create a separate function in a separate component. Both rely on Theme and props, so I decided not to abstract into utils.